src/components/Form/Form.tsx

```diff
@@ -15,7 +15,9 @@ const Form = ({ children, onInvalid, className, ...formAttrs }: FormProps): JSX.
 
   return (
     <form className={`bsds-form ${className || ''}`} onInvalid={handleInvalid} {...formAttrs}>
-      {children}
+      {React.Children.map(children, (child) => (
+        <div className="bsds-form-control">{child}</div>
```

Comment:
drop child

Reply:
Linters won't let me drop the children in both files 😢